### PR TITLE
[Doc] moving a platform to stuart

### LINKED ISSUE #87: Add "getting started" usage examples
Looking around the features, I'm really excited to give these tools a try in my own EDK2 projects, but I cannot figure out what steps I need to perform in order to integrate them into my codebase. 

[docs/using.md](https://github.com/tianocore/edk2-pytool-extensions/blob/master/docs/using.md) tantalizingly states:

> You are one settings file away from a great platform building experience.

But I cannot figure out what settings file it is referring to, how to create it, or where to put it.

Basically, it would be really helpful to have a getting started guide that walks through how to take an existing EDK2 project and adapt it to use these tools.

### PR DESCRIPTION
**Pull request by matthewfcarlson**
Adds a document that details how to port a platform to stuart
Fixes #87

### CODE REVIEW

**Comment by matthewfcarlson** [comment]
This is not ready but up for just general review to make sure it's on the right track in tone and style

**Comment by matthewfcarlson** [comment]
Another question is where should the repo go? Currently it's living on my personal github

**Comment by matthewfcarlson** [comment]
@spbrogan this is the tutorial I was mentioning

File: docs/usability/porting_a_platform.md [hunk 1]
```diff
@@ -0,0 +1,681 @@
+# Porting a platform to EDK2
```
Comment (matthewfcarlson) [inline docs/usability/porting_a_platform.md]: Maybe to EDK2 pytools?

Comment (corthon) [reply docs/usability/porting_a_platform.md]: Agreed.

**Comment by matthewfcarlson** [comment]
@corthon and @spbrogan I'd love to get this in today
@out0xb2 I'd love to get your feedback since this is something you've had some experience with recently :)

**Pull request merged**